pypesto/sampling/auto_correlation.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+from numba import jit
+
+
+@jit
+def sum_fast(nsimu: int,
```

Comment:
```suggestion
def _sum_fast(nsimu: int,
```
to indicate that this is only locally used here.